app.js

```diff
@@ -1,15 +1,15 @@
-var express = require('express')
-var app = express()
+var express = require('express');
+var app = express();
 
 app.get('/', function (req, res) {
-  res.send('Hello World!')
-})
+  res.send('Hello World!');
+});
 
 app.get('/index', (req, res) => {
   res.send('Hello index!');
 });
 
 app.listen(3000, function () {
-  console.log('Example app listening on port 3000!')
-})
+  console.log('Example app listening on port 3000!');
```

Comment:
Expected indentation of 4 spaces but found 2. (indent)
Unexpected console statement. (no-console)